jfrunit-core/pom.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<artifactId>jfrunit</artifactId>
+		<groupId>org.moditect.jfrunit</groupId>
+		<version>1.0.0.Alpha1</version>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>jfrunit-core</artifactId>
+
+	<name>JfrUnit</name>
+	<url>https://github.com/gunnarmorling/jfrunit/</url>
+	<description>A JUnit extension for asserting JDK Flight Recorder events</description>
+	<inceptionYear>2020</inceptionYear>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+		</dependency>
+		<dependency>
+            <groupId>org.junit.vintage</groupId>
+            <artifactId>junit-vintage-engine</artifactId>
+        </dependency>
+		<dependency>
+			<groupId>org.assertj</groupId>
+			<artifactId>assertj-core</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>io.quarkus</groupId>
+			<artifactId>quarkus-junit5</artifactId>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.spockframework</groupId>
+			<artifactId>spock-core</artifactId>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.spockframework</groupId>
+			<artifactId>spock-junit4</artifactId>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.codehaus.groovy</groupId>
+			<artifactId>groovy</artifactId>
+			<version>3.0.8</version>
+			<type>pom</type>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>${project.groupId}</groupId>
+			<artifactId>jfrunit-events-generator</artifactId>
+			<version>${project.version}</version>
+		</dependency>
```

Comment:
Can we add it on the process path instead (as e.g. described [here](https://mapstruct.org/documentation/dev/reference/html/#configuration-options))? I don't think we need to import any code from the generator?

That way, you also could pass the docs URL as an actual AP processor option (-A options in the example linked above), rather than as a system property.

Reply:
Wow, sure! this is exactly what I wanted to do but I couldn't pass the compilerArg like `<arg>-AjfrDocUrl=${jrfDocUrl}</arg>` your example helped a lot! thanks

Reply:
Yes, thanks, that looks great now.